js/src/components/stat.tsx

```diff
@@ -15,7 +15,7 @@ const StatsPane: React.FC<Prop> = ({ stats }) => (
             title={s.key}
             value={s.value.toFixed(0)}
             valueStyle={{
-              color: s.value === s.desired_value ? "green" : "red",
+              color: s.value >= s.desired_value ? "green" : "red",
```

Comment:
what does `desired_value` mean here?

Reply:
it's used to render different color on the page (the 24% can turn green if it is greater than, previously equal to, the desired value)